packages/builder/src/interface.ts

```diff
@@ -0,0 +1,417 @@
+/**
+ * Public interface for the builder package.
+ * This module exports only the types and functions that are part of the public API.
```

Comment:
It appears we modify interfaces here for use in the workspace in builder types, runner cell, etc. -- if this should *not* be imported by our workspace for relevant types, and instead should *only* be imported elsewhere with the extended interfaces, lets document that

Reply:
Good idea. Just added a comment that one should use only on or the other and changed all use locations accordingly.